### DISCUSSION TITLE
Map not showing in meeting list from Meeting Guide plugin

### DISCUSSION THREAD

**Discussion by lansingcentraloffice** [code4recovery/12-step-meeting-list#1294] [Q&A] [answered]
I am Belinda T, office manager for Lansing Central AA.
The meetings listed through the Meeting Guide Plugin are not showing the maps.
aalansingmi.org/meetings

I am not a coder.
I have rebuilt the site in Wordpress by myself and with the help from folks in the TIAA forum.
Today I pointed the url from where it had been on Squarespace to the new host, Siteground. (About 2 hours ago)
The plugin had been operating fine in the staging site under the temporary url.
At first the plugin wasn't working at all, but with the help of one of the admins in TIAA forum, we got the meetings to list at least. But when you click on a meeting the map doesn't upload.

Can someone help me get the maps to appear?

Deep gratitude,
Belinda T.

**Comment by anchovie91471** [comment]
Are you using mapbox or google as your maps key?

Can you check to make sure the key isn't empty in your 12 step meeting list settings?

**Reply by lansingcentraloffice** [reply]
Hi - Google is the maps key.

<img width="744" alt="Screen Shot 2023-12-30 at 8 26 06 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/155201720/34f4316d-ea71-457b-97d1-1800b0a71e80">

**Comment by dougie-wougie** [comment]
I am helping Belinda; I saw a key in the Google Maps field. I could validate that the key is valid. The site was recently moved from another host to SiteGround.

**Reply by lansingcentraloffice** [reply]
Good morning! You beat me to the reply. :)

**Reply by dougie-wougie** [reply]
All posts from here come into my RSS feeds, so I saw it this morning and wanted to be helpful. I am curious why it does not work. I think moving from one hosting platform to another is the issue; if that is true and a new API key is required, I favor MapBox over Google, but that is just a preference.

**Comment by brianw-area24** [comment]
Thank you for posting the image of the settings screen, that really helped. Although the "User Interface Display" appears to be legacy, when I look at your site, I'm seeing TSML-UI as the interface. I'd suggest changing this setting to TSML-UI, then changing it back to Legacy UI. I'd like to see if that makes any difference.

TSML UI does not support anything other than Mapbox, but the Legacy UI should allow it to use the Google maps.

**Reply by lansingcentraloffice** [reply]
Thanks Brian,
Before I change it to TSML-UI I want to invite you to look at the site on mobile.
It seems to pull the maps just fine on mobile.


B

**Comment by lansingcentraloffice** [comment]
OK, I switched it to TSML_UI
No change. But you can see the map if you click "Get Directions"

<img width="1156" alt="Screen Shot 2023-12-30 at 2 09 28 PM" src="https://github.com/code4recovery/12-step-meeting-list/assets/155201720/67b649e0-bff2-4118-b354-ce2e4edb0904">

**Reply by brianw-area24** [reply]
OK, now change it back to Legacy UI, does it change?

**Comment by lansingcentraloffice** [comment]
Nope Sent from my iPhoneOn Dec 30, 2023, at 2:28 PM, brianw-area24 ***@***.***> wrote:﻿
OK, now change it back to Legacy UI, does it change?

**Reply by brianw-area24** [reply]
OK, I'm not sure why we're seeing the TSML-UI, when you have Legacy UI selected in settings. We talked about this issue during our meeting earlier today, so I'll take it back to the group and see if anyone else has anything.

**Comment by lansingcentraloffice** [comment]
Dougie introduced that code. It was the only thing that seems to make the listing show at all. I moved my site from Square space to WordPress yesterday. In switching the name server it seemed that the plug-in was no longer functioning. So that's the reason that you see That code introduced on that page. Sent from my iPhoneOn Dec 30, 2023, at 4:12 PM, brianw-area24 ***@***.***> wrote:﻿
OK, I'm not sure why we're seeing the TSML-UI, when you have Legacy UI selected in settings. We talked about this issue during our meeting earlier today, so I'll take it back to the group and see if anyone else has anything.

**Comment by dougie-wougie** [comment]
Thanks, folks Belinda; I copied my shortcut [tsml_ui], and it should be the other. I am not sure what the shortcut should be. Possibly [tsml]? I looked at the FAQ, and it is not listed, or I do not see it.

d

**Reply by brianw-area24** [reply]
Is the meetings page we're looking (http://aalansingmi.org/meetings/) a regular page/post that just has a shortcode on it, or is the page that the TSML pluging should be creating? From what @dougie-wougie said, I'm thinking it's a page with the shortcode. If this is the case, then it makes sense of what we're seeing.

If it is a separate page, I'd like you to do the following:
1. Delete that page/post, it shouldn't be needed, and would be easy to recreate it if necessary
2. Go to Settings->Permalinks in the WordPress dashboard
3. Save the permalinks page, you don't need to make any changes, just save it
4. See if you have a meeting page, and what it works like

**Reply by lansingcentraloffice** [reply]
Brian, I actually can't remember, so I'm going to give that a try right now.

Belinda

**Reply by lansingcentraloffice** [reply]
OK. Thanks for that.
I did what you specified above.

1) I don't see a new page created under pages in the Wordpress Dashboard. But I added "Meetings" in the menu from Meetings selection.
2) When you click a meeting listing  it starts to try to load the map then stops and shows an error.
![IMG_5588](https://github.com/code4recovery/12-step-meeting-list/assets/155201720/d55daaa2-41f5-4c36-80c2-48353bef9c38)

**Reply by lansingcentraloffice** [reply]
also... the listing starts at 6:30 a.m. and not the most recent time as it had before.
Previously it would have said how many meetings are going on right now and show the list from the current time through the rest of the day.

**Comment by joshreisner** [comment, marked as answer]
hi @lansingcentraloffice looks like your google maps key is not activated, it says:

```
Google Maps JavaScript API error: ApiNotActivatedMapError
https://developers.google.com/maps/documentation/javascript/error-messages#api-not-activated-map-error
```

rather than activating it though I recommend you switch to Mapbox at Mapbox.com and get a free Mapbox Access Token - this will allow you to switch back to TSML UI. this will address your comment that:

> Previously it would have said how many meetings are going on right now and show the list from the current time through the rest of the day.

**Reply by lansingcentraloffice** [reply]
Aaah. Thank you.
I will give that a try.

**Reply by lansingcentraloffice** [reply]
SUCCESS!

Much gratitude for all the generous help here!

Belinda T.
